src/Datasource/EntityInterface.php

```diff
@@ -252,17 +252,20 @@ public function extract(array $fields, bool $onlyDirty = false): array;
     public function clean(): void;
 
     /**
-     * Returns whether or not this entity has already been persisted.
-     * This method can return null in the case there is no prior information on
-     * the status of this entity.
+     * Set the status of this entity.
+     *
+     * Using `true` means that the entity has not been persisted in the database,
+     * `false` that it already is.
      *
-     * If called with a boolean, this method will set the status of this instance.
-     * Using `true` means that the instance has not been persisted in the database, `false`
-     * that it already is.
+     * @param bool $new Indicate whether or not this entity has been persisted.
+     * @return $this
+     */
+    public function setNew(bool $new);
```

Comment:
Default to `true` like `setDirty()`?

Reply:
Usually entity is new by default and in most cases this method is called with `false` argument. So default `true` doesn't feel right to me. So `setAccess()` does not have a default value so `setNew()` not having one is fine IMO.